cloudify-manager-worker/templates/k8s_secrets.yml

```diff
@@ -0,0 +1,109 @@
+apiVersion: ENC[AES256_GCM,data:cUQ=,iv:hy/ZdP1le16X3DmD82wU1ESBUsmlVfWwtgDrl6QQyfs=,tag:tjK4GQ0ZpbkT4jsbJR6ICA==,type:str]
```

Comment:
We should not use things like this inside the helm chart. If you want to encrypt something using 3rd-party tool - you need to do it in override values outside chart.
Also this is a public repo, even if we have secrets encrypted some information from this file should not be public-available like AWS KMS ARN, which includes AWS account ID (non-secret but can be used for reduce attack surface).